sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,9 +1,26 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-10-08)
 
+### New Features
+
+- Added NOTICE.txt for 3rd party dependencies
+
```

Comment:
This isn't really a **feature** of the SDK though from an API usage perspective. Remove (or move to "other changes" if you really want to keep it).
```suggestion
```